### PR TITLE
Windows fixes for examples tests

### PR DESCRIPTION
**Pull request by gregmagolan**
MANIFEST fixes for examples tests for Windows

/cc @alexeagle

### CODE REVIEW

File: examples/es5_output/es5_output_test.sh [hunk 1]
```diff
@@ -1,40 +1,69 @@
...
 if [[ "$BAR_JS" != *"require(\"build_bazel_rules_typescript/examples/foo\")"* ]]; then
   echo "Expected bar.js to require named module foo, but was"
   echo "$BAR_JS"
   exit 1
 fi
 
 # should give a name to required modules from other compilation unit
-readonly FOO_JS=$(cat $TEST_SRCDIR/build_bazel_rules_typescript/examples/bar.js)
-if [[ "$FOO_JS" != *"require(\"some-lib/library\")"* ]]; then
+if [[ "$BAR_JS" != *"require(\"some-lib/library\")"* ]]; then
```
Comment (alexeagle) [inline examples/es5_output/es5_output_test.sh]: why did this change from FOO to BAR? seems like a mistake?

Comment (gregmagolan) [reply examples/es5_output/es5_output_test.sh]: the original was wrong. code was actually pointing to bar.js: `readonly FOO_JS=$(cat $TEST_SRCDIR/build_bazel_rules_typescript/examples/bar.js)`

**Comment by gregmagolan** [comment]
@mprobst Can you review when you have some time. This fixes the last CI failures on Windows.

**Comment by gregmagolan** [comment]
@mprobst Please review. This fixes the last CI failures on Windows. Thanks

**Comment by alexeagle** [comment]
I'll find some other googler to review. But looking at this again, it seems like we have way too much duplication of the logic to lookup a file possibly through the manifest. Any thoughts how we could make a shared .sh file that we source and call a function? Should be shared with e.g. https://github.com/bazelbuild/rules_nodejs/blob/master/internal/node/node_launcher.sh#L106 as well

**Comment by gregmagolan** [comment]
https://github.com/bazelbuild/rules_nodejs/pull/207 adds a bash function that can be used to resolve runfiles. Will reduce the duplication here once that lands.

**Pull request closed without merging**